assembl/locale/de/LC_MESSAGES/assembl.po

```diff
@@ -5175,11 +5175,14 @@ msgid "No discussion found for slug=%s"
 msgstr "Keine Diskussion gefunden für slug=%s"
 
 #~ msgid ""
-#~ "Our server has encountered a problem."
-#~ " The page you have requested is "
-#~ "not accesible."
 #~ msgstr ""
 
 #~ msgid "We apologise for the inconvenience!"
 #~ msgstr ""
 
+#~ msgid "We apologize for the inconvenience!"
```

Comment:
You can remove these 2 as well, they are old code and shouldn't be in the POT file